StellarDotnetSdk/Operations/InvokeHostFunctionOperation.cs

```diff
@@ -108,22 +108,25 @@ public class CreateContractOperation : InvokeHostFunctionOperation
     /// <summary>
     ///     Creates a new <c>CreateContractOperation</c> using the provided address and salt.
     /// </summary>
-    /// <param name="wasmId"></param>
+    /// <param name="wasmHash">
+    ///     (aka Wasm ID or hash ID or just hash), a hex-encoded string of the unique identifier of a
```

Comment:
Is this not the hex-encoded Wasm ID?

Reply:
This TODO is supposed for later. If you check the link, you can see the properly is not included in the list, but it's there in the example.
I thought maybe better to leave it there as a note.

Reply:
Yes, it's the hex-encoded ID. Previously, I made it to expect a base64 string without realizing in Soroban CLI and other places, a hex-encoded ID is required.